packages/auto-instrumentations-node/src/instrumentations/index.ts

```diff
@@ -48,11 +48,6 @@ export function getCiscoNodeAutoInstrumentations(
   // TODO: once we have package per instrumentation, we will have to remove this
   //       because we will want to give the user option to config every instrumentation by itself.
   const options = _configDefaultOptions(userOptions);
-
-  if (options.debug) {
-    diag.setLogger(new DiagConsoleLogger(), DiagLogLevel.DEBUG);
-  }
```

Comment:
no logs & debug in this package?

Reply:
Yes, this is for OTel native users, they can set up they're own logger ;)